cloudhub/modules/ROOT/pages/app-migration.adoc

```diff
@@ -46,6 +46,13 @@ If you're using `anypoint.dns.net` without a TLS context, the *Ingress* tab does
 .. Select whether you want to provide internal or external access to your endpoint.
 . Follow the steps to xref:cloudhub-2::ch2-deploy.adoc[deploy an application to CloudHub 2.0].
 
+[WARNING]
+====
+If you use a different name for the CloudHub 2.0 application, make sure you input the CloudHub application name in the subdomain field of the endpoint configuration within the *Ingress* tab settings. Failing to configure the correct URLs can result in traffic not reaching the CloudHub 2.0 application during xref:app-migration.adoc#next-steps-after-application-upgrade[traffic switching].
```

Comment:
```suggestion
If you use a different name for the CloudHub 2.0 application, make sure you put the CloudHub application name in the *Ingress* tab settings in the subdomain field of the endpoint configuration. If you configure URLs incorrectly, traffic doesn't reach the CloudHub 2.0 application during xref:app-migration.adoc#next-steps-after-application-upgrade[traffic switching].
```

Reply:
Simplified language, active vs. passive, and putting the procedural part in the order they'd encounter it. Please verify that is correct. :)